apps/src/music/views/music-play-view.module.scss

```diff
@@ -37,23 +51,48 @@
     .musicPlayViewPlaySection {
       display: flex;
       align-items: center;
-      gap: 0.5rem;
+      gap: 1rem;
+
+      input[type=range] {
```

Comment:
Reference: https://css-tricks.com/styling-cross-browser-compatible-range-inputs-css/
Style range inputs so that styling is consistent across Firefox, Edge, Safari, and Chrome.

Reply:
Cool stuff!

Reply:
Slick, need this for AI Chat too!